database/migrations/2024_08_26_070304_importers_requests_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration {
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('importers_requests', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger('apps_id')->unsigned();
+            $table->bigInteger('users_id')->unsigned();
+            $table->bigInteger('companies_id')->unsigned();
+            $table->bigInteger('regions_id')->unsigned();
+            $table->bigInteger('companies_branches_id')->unsigned();
+            $table->bigInteger('filesystem_id')->unsigned();
```

Comment:
@FredPeal since you already have the header wouldn't it be cool to put it here ?

Reply:
@FredPeal also add index to all _id